app/src/test/java/com/example/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -13,46 +13,32 @@ import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.rules.TestRule
 
-
-
 class InstrumentDataIntegrationTest {
 
     @get:Rule
-    var rule: TestRule =  InstantTaskExecutorRule()
-    lateinit var mvm:MainViewModel
-
+    var rule: TestRule = InstantTaskExecutorRule()
+    lateinit var mvm: MainViewModel
     var instrumentService = mockk<InstrumentService>()
 
-
     @Test
-    fun TestInstrumentListForResults() {
-
+    internal fun TestInstrumentListForResults() {
```

Comment:
One more example of a function that should be declared as internal, to prevent access outside of the module.